### ISSUE TITLE
A11y_Visual Studio Code_Accessing The  Extension  bar options_ScreenReader: "JAWS" is not announcing the Count of extensions available

### ISSUE THREAD

**Issue by v-kanve** [microsoft/vscode#122898]
### Actual Result:
"JAWS" is not announcing the Count of extensions available

### Expected Result: ​
"JAWS" should announce the Count of extensions available. First it should announce about the button details and then the count of extensions available. Ex. 0 extensions available or 5 extensions available

GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#JAWS;#ScreenReader;#WCAG1.3.1;#Win32;#Element:Button;#AINotInScope;#Win10;#VisualStudioCode-Web-Apr2021;

### Environment Details:​
Application Name: Visual Studio Code
Screen Reader: JAWS
Standard Violated:WCAG1.3.1

### Repro Steps:​
1. Turn on JAWS
2. Open Visual Studio Code editor
3. Navigate to the extensions in the sidebar options using "Ctrl+Shift+X" key
4. Tab till "Installed", "Popular" and "Recommended"
5. Verify whether JAWS is reading the count of extensions or not

### User Impact:
Screen Reader users will face difficulty while navigation if JAWS does not announce the number of extensions present. They will not know the count of extensions available.

### Attachment:
[A11y_Visual Studio Code_Accessing The  Extension  bar options_ScreenReader JAWS is not announcing the Count of extensions available.zip](https://github.com/microsoft/vscode/files/6419236/A11y_Visual.Studio.Code_Accessing.The.Extension.bar.options_ScreenReader.JAWS.is.not.announcing.the.Count.of.extensions.available.zip)

**Comment by sandy081** [comment]
@isidorn Assigning to you to use your expertise

**Comment by isidorn** [comment]
Once the user moves the focus to the list the screen reader will actually read how many items there are in total since we correctly set the `aria-setsize` on the list.
This should not be informed to the user while he or she is in the input box, since most of the time there will be too many results and this information is not useful to provide at that point. Only once the focus is moved to the list of extensions

Closing as designed

![Screenshot 2021-06-11 at 15 09 15](https://user-images.githubusercontent.com/1926584/121691229-0d936c00-cac7-11eb-94ff-babffa7a7403.png)
